packages/@o3r/eslint-config/schematics/ng-add/index.ts

```diff
@@ -109,7 +109,7 @@ const addHarmonizeScript = (): Rule => {
       return tree;
     }
     const postInstall = rootPackageJsonObject.scripts.postinstall;
-    rootPackageJsonObject.scripts.harmonize = `eslint '**/package.json' ${isYarnPackageManager ? '.yarnrc.yml ' : ''}--quiet --fix --no-error-on-unmatched-pattern`;
+    rootPackageJsonObject.scripts.harmonize = `eslint "**/package.json" ${isYarnPackageManager ? '.yarnrc.yml ' : ''}--quiet --fix --no-error-on-unmatched-pattern`;
```

Comment:
don't you need to escape the doublequotes ?
```suggestion
    rootPackageJsonObject.scripts.harmonize = `eslint \\"**/package.json\\" ${isYarnPackageManager ? '.yarnrc.yml ' : ''}--quiet --fix --no-error-on-unmatched-pattern`;
```

Reply:
It is not already escaped by the `JSON.stringify()`?